pkg/build/builder/cmd/builder.go

```diff
@@ -242,3 +295,30 @@ func RunDockerBuild(out io.Writer) error {
 func RunS2IBuild(out io.Writer) error {
 	return runBuild(out, s2iBuilder{})
 }
+
+// RunGitClone performs a git clone using the build defined in the environment
+func RunGitClone(out io.Writer) error {
+	cfg, err := newBuilderConfigFromEnvironment(out, false)
+	if err != nil {
+		return err
+	}
+	return cfg.clone()
+}
+
+// RunManageDockerfile manipulates the dockerfile for docker builds
```

Comment:
Maybe a better name would be RewriteDockerfile ?
ManageDockerfile doesn't tell me much about what this does